pkg/commands/commands.go

```diff
@@ -1,12 +1,12 @@
-package controllers
+package commands
 
 import (
 	"fmt"
 
-	cloudstorages "github.com/st-tech/gatling-operator/controllers/cloudstorages"
+	cloudstorages "github.com/st-tech/gatling-operator/pkg/cloudstorages"
 )
 
-func getGatlingWaiterCommand(parallelism *int32, gatlingNamespace string, gatlingName string) string {
+func GetGatlingWaiterCommand(parallelism *int32, gatlingNamespace string, gatlingName string) string {
```

Comment:
Why did you change from lowerCamelCase to UpperCamelCase?

Reply:
Starting from Capital means allowing public access, which is needed as the command functions are move on to pks and set different package name like `commands` to make it shared library

Reply:
In other other, command functions are no longer accessible in local scope

Reply:
I got it! thanks.